aws/v19.1.0/release.diff

```diff
@@ -0,0 +1,179 @@
+    - cilium                                                         - cilium
+    - coredns                                                        - coredns
+  - name: k8s-dns-node-cache-app                               |     name: observability-bundle
+    version: 2.4.0                                             |     version: 0.7.3
+    dependsOn:                                                 |   - dependsOn:
+    - aws-cloud-controller-manager                                   - aws-cloud-controller-manager
+    - cilium                                                         - cilium
+    - coredns                                                        - coredns
+    - prometheus-operator-crd                                        - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd                                    - vertical-pod-autoscaler-crd
+  - name: prometheus-blackbox-exporter                         |     name: k8s-dns-node-cache-app
+    version: 0.3.2                                             |     version: 2.4.0
+    dependsOn:                                                 |   - dependsOn:
+    - aws-cloud-controller-manager                                   - aws-cloud-controller-manager
+    - cilium                                                         - cilium
+    - coredns                                                        - coredns
+    - vertical-pod-autoscaler-crd                                    - vertical-pod-autoscaler-crd
+  - name: cilium-servicemonitors                               |     name: prometheus-blackbox-exporter
+    version: 0.1.1                                             |     version: 0.3.2
+    dependsOn:                                                 |   - dependsOn:
+    - prometheus-operator-crd                                        - prometheus-operator-crd
+  - name: irsa-servicemonitors                                 |     name: cilium-servicemonitors
+    version: 0.0.1                                             |     version: 0.1.2
+    dependsOn:                                                 |   - dependsOn:
+    - prometheus-operator-crd                                        - prometheus-operator-crd
+                                                               >     name: irsa-servicemonitors
+                                                               >     version: 0.0.1
+  components:                                                      components:
+  - name: app-operator                                             - name: app-operator
+    version: 6.7.0                                             |     version: 6.8.0
+  - name: aws-operator                                             - name: aws-operator
+    releaseOperatorDeploy: true                                < 
+    version: 14.17.1                                           < 
+    reference: 14.17.1-patch3                                        reference: 14.17.1-patch3
```

Comment:
This looks weird, the reference is still 14.17.1-patch3 but the version has changed to 14.21.0